index.md

```diff
@@ -15,7 +15,7 @@ These lessons will start you on a path towards using these resources effectively
 
 > ## Prerequisites {.prereq}
 >
-> This lesson guides you through the basics of filesystems and the
+> This lesson guides you through the basics of file systems and the
```

Comment:
You're right, we're inconsistent - 17 uses of 'filesystem' (one word) in our lessons, 12 uses of 'file system' (two words).  In the shell novice lesson, though, it's 11 to 12, so yes, make the change.